presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -1627,6 +1660,10 @@ private static void assertRowExpressionEvaluationEquals(Object left, Object righ
             assertTrue(left instanceof RowExpression);
             // assertEquals(((RowExpression) left).getType(), ((RowExpression) right).getType());
             if (left instanceof ConstantExpression) {
+                if (isRemovableCast(right)) {
```

Comment:
The reason we need this is that, `ExpressionInterpreter` can have unnecessary cast due to lack of type information (for example  `CAST ( 1 AS BGINT)`).